safe-global/safe-react-apps
# feat(safe-claiming-app): Dark theme

## What it solves
Dark mode for Claiming app + widgets

Depends on https://github.com/safe-global/web-core/pull/1219 to be tested

## How this PR fixes it
Reads a dark theme flag from the appUrl with a fallback to the system preference

## How to test it
Two ways to test the dark mode:
  - Set your system preferences to dark
  - Needs to run https://github.com/safe-global/web-core/pull/1219 and this branch locally.
Observe the Governance widgets load according to the web-core app theme.
The Claiming app has dark mode applied.


## Screenshots
<img width="1035" alt="Screenshot 2022-11-22 at 13 19 26" src="https://user-images.githubusercontent.com/32431609/203312147-037db198-eaa0-49fa-8255-52b98951769e.png">
